lib/stat_generator.rb

```diff
@@ -48,19 +48,74 @@ def total_ties
       game.away_goals == game.home_goals
     end
   end
-
+  
   def percentage_ties
     (total_ties / count_of_games.to_f).round(2)
+  end 
+  
+  def season_names
```

Comment:
This accomplishes the same as the `seasons` method. I prefer this one but okay with keeping either. One of them will need to go before merging though.